orcid-solr-web/src/main/resources/log4j.xml

```diff
@@ -39,7 +39,7 @@
 	<!-- Root Logger -->
 	<root>
 		<level value="DEBUG" />
-		<!-- <appender-ref ref="CONSOLE" /> -->
```

Comment:
Hey @TomDemeranville, shouldnt your rollback this?

Reply:
Good catch!